ts-cloudfs/package.json

```diff
@@ -17,6 +17,7 @@
     "typescript": "^4.8.4"
   },
   "dependencies": {
-    "express": "^4.17.2"
+    "express": "^4.17.2",
+    "ts-node": "^10.9.1"
```

Comment:
nit: this should be in `devDependencies`

Reply:
will include these comments changes in the testing prs since this just got merged